lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -232,24 +232,54 @@ uint32_t SessionState::get_current_version() {
 }
 
 void SessionState::set_current_version(
-    int new_session_version, SessionStateUpdateCriteria* session_uc) {
-  current_version_ = new_session_version;
-  if (session_uc) {
-    session_uc->is_current_version_updated = true;
-    session_uc->updated_current_version    = new_session_version;
-  }
-  MLOG(MINFO) << " Current version is " << get_current_version();
+    uint32_t new_session_version, SessionStateUpdateCriteria& session_uc) {
+  current_version_                      = new_session_version;
+  session_uc.is_current_version_updated = true;
+  session_uc.updated_current_version    = new_session_version;
+  MLOG(MDEBUG) << " Current version is " << get_current_version();
 }
+
 /* Add PDR rule to this rules session list */
-void SessionState::insert_pdr(SetGroupPDR* rule) {
-  PdrList_.push_back(*rule);
+void SessionState::insert_pdr(
+    SetGroupPDR* rule, SessionStateUpdateCriteria& session_uc, bool crit_add) {
```

Comment:
`SetGroupPDR* rule, SessionStateUpdateCriteria& session_uc, bool crit_add` ->
`SetGroupPDR* rule, bool crit_add, SessionStateUpdateCriteria* session_uc)`

Reply:
Modified the function parameter to pointer instead of reference, order of parameters as suggested.